app/workers/image_callback_worker.rb

```diff
@@ -17,18 +19,21 @@ def perform(_sqs_msg, job)
     image.content_type = mime_type || job['format']
 
     if !job['downloaded']
-      image.status = Image::NOTFOUND
+      image.status = NOTFOUND
     elsif !job['valid']
-      image.status = Image::INVALID
+      image.status = INVALID
     elsif !job['resized']
-      image.status = Image::FAILED
+      image.status = FAILED
     else
       image.upload_path = nil
-      image.status = Image::COMPLETE
+      image.status = COMPLETE
     end
 
     Shoryuken.logger.info("Updating #{job['type']}[#{image.id}]: status => #{image.status}")
     image.save!
+    # announce the audio changes using the story as resource -- or do we want series?
+    announce(:image, :update, id: image.story.id, resource: image.story)
```

Comment:
But images don't have a story, right?  Or at least not all of them do?  (This callback is shared by series_images, story_images, account_images, etc).

This should announce on the parent channel ... but we probably only care about series and story here.  So maybe detect if the image is one of those, and fire off the announcement for just those?